about.html

```diff
@@ -0,0 +1,112 @@
+      <ul class="nav-menu">
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a>Program</a>
+        </li>
+        <li>
+          <a>Join</a>
+        </li>
+        <li>
+          <a>Sponsor</a>
+        </li>
+        <li>
+          <a>News</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="links-container">
+    <ul>
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a>Program</a>
+      </li>
+      <li>
+        <a>Join</a>
+      </li>
+      <li>
+        <a>Sponsor</a>
+      </li>
+      <li>
+        <a>News</a>
+      </li>
+      <li>
+        <button type="button">CC Campaign</button>
+      </li>
+    </ul>
+  </div>
+</nav>
+<!-- header -->
+<header class="header">
+  <div class="about-header-container">
+    <h1 class="about-intro">10-Gano Courses</h1>
+    <p class="about-intro-paragraph">Learn at the comfort of your own home. 10-Gano website is the world's best platform for whoever is willling to develop their skillset and advance career.</p>
+    <p class="about-contact-p">Please contact us per Email for any further questions about latest Courses</p>
+    <p class="about-email">info@10-gano.com</p>
+    <h5></h5>
+  </div>
+</header>
+<!-- logo -->
+<section class="logo-section">
+    <h6 class="logo-section-head">Creative Comons Global Summit 2015 Logo</h6>
+    <hr class="main-program-hr">
+    <p class="logo-section-desc">The logo of 10-Gano Platform for most high demanding courses was decided through the logo competition from 8. June to 7. July.</p>
+    <img src="assets/10-gano-high-resolution-logo-black-on-transparent-background.png" alt="logo" class="logo">
+</section>
+<!-- past summits section -->
+<section class="past-summits-sections">
+    <h3 class="past-summits-heading">See the past 10-Gano events</h3>
+    <hr class="main-program-hr">
+    <p class="past-summits-desc">Take a look at the past two events which took place in Buenos Aires and in Warsaw</p>
+    <div class="past-summits-images-container">
+      <div class="img-container">
+        <img src="assets/image03.png" alt="summit">
+        <div class="img-content-container">
+          <h6 class="summit-year">2011</h6>
+          <h6 class="summit-desc">CC Global Summit 2011 in Warsaw</h6>
+        </div>
+      </div>
+      <div class="img-container">
+        <img src="assets/image03.png" alt="summit">
+        <div class="img-content-container">
+          <h6 class="summit-year">2011</h6>
+          <h6 class="summit-desc">CC Global Summit 2011 in Warsaw</h6>
+        </div>
+      </div>
+    </div>
+</section>
+<!-- footer -->
+<footer class="about-footer">
+  <div class="footer-logo-container"><img src="assets/10-gano-high-resolution-logo-black-on-transparent-background.png" alt="logo"></div>
+  <div class="copyright-container">
+    <p class="copyright">10-Gano Platform for courses. Some Rights Reserved.</p>
+  </div>
+</footer>
```

Comment:
Please kindly fix the floating image in the `mobile version` of the `About` section's `footer` and add the footer accordingly 

Your Design (Footer) 👇 
![image](https://user-images.githubusercontent.com/78251927/225746051-5fee5a71-f052-4274-8b1f-423b16f41b19.png)

Template 👇 
![image](https://user-images.githubusercontent.com/78251927/225746542-b31fbb64-0802-4588-bc96-8c6aa2edd703.png)